dotCMS/src/integration-test/java/com/dotmarketing/startup/StartupTasksExecutorTest.java

```diff
@@ -0,0 +1,54 @@
+package com.dotmarketing.startup;
+
+import static org.junit.Assert.assertEquals;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(DataProviderRunner.class)
+public class StartupTasksExecutorTest {
+
+    @DataProvider
+    public static Object [] testCases() {
+        return new TestCase []{
+                new TestCase("201009", "Task201009TestMethod"),
+                new TestCase("2010091", "Task2010091TestMethod"),
+                new TestCase("05030", "Task05030TestMethod")
+        };
+    }
+
+    private static class TestCase{
+        String taskId;
+        String taskName;
+
+        public TestCase(final String taskId, final String taskName){
+            this.taskId = taskId;
+            this.taskName = taskName;
+        }
+    }
+
+    @BeforeClass
+    public static void prepare() throws Exception{
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354433)